l1-contracts/bootstrap.sh

```diff
@@ -408,6 +408,21 @@ function release_git_push {
   echo "Release complete ($tag_name) on branch $branch_name."
 }
 
+function coverage {
+  echo_header "l1-contracts coverage"
+  local report_type=${1:-}
+  local fancy=${2:-}
+  if [ "$report_type" = "lcov" ]; then
+    forge coverage --no-match-coverage "(test|script|mock|generated)" --report lcov
+    if [ "$fancy" = "fancy" ]; then
+      mkdir -p coverage
+      genhtml lcov.info --branch-coverage --output-dir coverage
```

Comment:
bootstrap / dev image install for genhtml is missing

Reply:
(cd coverage && python3 -m http.server 8080) could be added to bootstrap cov-serve?